core/scrollbar.js

```diff
@@ -196,7 +201,9 @@ Blockly.Scrollbar = function(workspace, horizontal, opt_pair, opt_class) {
   this.createDom_(opt_class);
 
   /**
-   * The upper left corner of the scrollbar's svg group.
+   * The upper left corner of the scrollbar's svg group in CSS pixels relative
```

Comment:
The comment on prototype.origin_ below says it's the position of the "scrollbar SVG" relatively to the injection div, this says it's the position of the "scrollbar's svg group" relative to the origin. Is that the origin below? If so why is this ever something other than 0?